Xamarin.PropertyEditing.Windows/Themes/Resources.xaml

```diff
@@ -1113,6 +1190,150 @@
 		</Setter>
 	</Style>
 
+	<Style x:Key="BrushChoiceControlItem" BasedOn="{StaticResource {x:Type RadioButton}}" TargetType="RadioButton">
```

Comment:
Can we name this more specifically? It's meant for the material picker right? It sounds like its for selecting the brush type.

Reply:
Sure.